Makefile

```diff
@@ -57,3 +57,6 @@ ci-all: install-platformio
 
 ci-basic:
 	$(platformio) ci --board=$(board) --lib="." examples/HX711_basic_example --verbose
+
+clean:
+	platformio run -t clean
```

Comment:
$(...)? ;)

Reply:
Thanks for spotting this, @rohlan! I just pushed an update to the same branch, but @bogde merged this PR already. So, it will probably become part of eventual future updates.

Reply:
Made #126 from it. Thanks again.